content/swift/concepts/arrays/terms/randomElement/randomElement.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: '.randomElement()'
+Description: 'Returns a random element from of arrary.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The ** `.randomElement()` method will randomly return an element from an  array. If the array is empty, it will return nil.
+
+## Syntax
+
+```swift 
+arrayName.randomElement()
+
+```
+
```

Comment:
Generally we try to include a sentence or two after the syntax block explaining the parameters, or functionality of the method, etc